workflow/envs/savana.yaml

```
@@ -0,0 +1,14 @@
+  - bioconda
+  - nodefaults
+dependencies:
+  - bcftools
+  - pybedtools==0.9.0
+  - pysam==0.20.0
+  - cyvcf2==0.30.16
+  - scikit-learn==1.2.2
+  - pandas==2.0.0
+  - matplotlib==3.7.1
+  - pip:
+      - git+https://github.com/johanneskoester/savana.git@fix/empty-calls
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)